tools/dotnet-scaffold-aspire/Commands/StorageCommand.cs

```diff
@@ -1,51 +1,232 @@
+        webApiSettings.AddSettings("workspace", workspaceSettings);
+        var codeService = new CodeService(webApiSettings, _logger);
+
+        CodeChangeOptions options = new()
+        {
+            IsMinimalApp = await ProjectModifierHelper.IsMinimalApp(codeService),
+            UsingTopLevelsStatements = await ProjectModifierHelper.IsUsingTopLevelStatements(codeService)
+        };
+
+        config = EditConfigForApiService(config, options, commandSettings.Type);
+        var projectModifier = new ProjectModifier(
+        _environmentService,
+        webApiSettings,
+        codeService,
+        _logger,
+        options,
+        config);
+        return await projectModifier.RunAsync();
+    }
+
+    internal CodeModifierConfig? EditConfigForAppHost(CodeModifierConfig? configToEdit, CodeChangeOptions codeChangeOptions, string? projectName, string storageType)
+    {
+        if (configToEdit is null)
+        {
+            return null;
+        }
+
+        var programCsFile = configToEdit.Files?.FirstOrDefault(x => !string.IsNullOrEmpty(x.FileName) && x.FileName.Equals("Program.cs", StringComparison.OrdinalIgnoreCase));
+        if (programCsFile != null &&
+            programCsFile.Methods != null &&
+            programCsFile.Methods.Count != 0 &&
+            StorageConstants.StoragePropertiesDict.TryGetValue(storageType, out var storageProperties))
+        {
+            var globalMethod = programCsFile.Methods.Where(x => x.Key.Equals("Global", StringComparison.OrdinalIgnoreCase)).First().Value;
+            //to inject correct storage variables in 'var {0} = storage.{1}("{0}")'
+            var addAzureStorageChange = globalMethod?.CodeChanges?.FirstOrDefault(x => !string.IsNullOrEmpty(x.InsertAfter) && x.InsertAfter.Contains("builder.AddAzureStorage"));
+            var addProjectChange = globalMethod?.CodeChanges?.FirstOrDefault(x => !string.IsNullOrEmpty(x.Parent) && x.Parent.Contains("builder.AddProject<{0}>"));
+            //apply changes to addAzureStorageCommand
+            if (!codeChangeOptions.UsingTopLevelsStatements && addAzureStorageChange != null)
+            {
+                addAzureStorageChange = DocumentBuilder.AddLeadingTriviaSpaces(addAzureStorageChange, spaces: 12);
+            }
+
+            if (addAzureStorageChange != null && !string.IsNullOrEmpty(addAzureStorageChange.Block) && storageProperties != null)
+            {
+                //update the parent value with the project name inserted.
+                addAzureStorageChange.Block = string.Format(addAzureStorageChange.Block, storageProperties.VariableName, storageProperties.AddMethodName);
+            }
+
+            //apply changes to addProjectChange
+            if (!codeChangeOptions.UsingTopLevelsStatements && addProjectChange != null)
+            {
+                addProjectChange = DocumentBuilder.AddLeadingTriviaSpaces(addProjectChange, spaces: 12);
+            }
+
+            if (addProjectChange != null &&
```

Comment:
nit: There's a blend of `!= null` and `is null` in the changes here, we should be consistent (preferably with `is`/`is not` unless `==`/`!=` is truly correct in a particular scenario)

Reply:
yep will do a full sweep soon, I'll just get the ones in this PR currently tyty

Reply:
Yeah i was just referring to this change since I noticed it in the same + block. Don't worry about other places at the moment. We can use analyzers and such.